docs/reference/dockerd.md

```diff
@@ -1076,11 +1076,11 @@ The following is a full example of the allowed configuration options on Linux:
   "builder": {
     "gc": {
       "enabled": true,
-      "defaultKeepStorage": "10GB",
+      "reservedSpace": "10GB",
       "policy": [
-        { "keepStorage": "10GB", "filter": ["unused-for=2200h"] },
-        { "keepStorage": "50GB", "filter": ["unused-for=3300h"] },
-        { "keepStorage": "100GB", "all": true }
+        { "maxUsedSpace": "512MB", "keepDuration": "48h", "filter": [ "type=source.local" ] },
+        { "reservedSpace": "10GB", "maxUsedSpace": "100GB", "keepDuration": "1440h" },
+        { "reservedSpace": "50GB", "minFreeSpace": "20GB", "maxUsedSpace": "200GB", "all": true }
       ]
```

Comment:
Perhaps @crazy-max or @jsternberg can help review 😅 

These options always confuse me (e.g. in this case `maxUsedSpace` being 512MB, but other options having a larger size).

> The filter field in daemon.json uses a single `=` to assign filters, whereas builder uses `==`, for example: `type==source.git.checkout`

Is that something that should be aligned? I also recall  the "single filter allowed" part on the daemon, and was confused by that (i.e., the daemon disallows multiple filters, but I couldn't find why it added that restriction); https://github.com/moby/moby/issues/46864

Reply:
I think alignment would be great. I see no reason that the formats should be separate. Also for the size values.